src/Contracts/ConfigManager.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Spatie\Ignition\Contracts;
+
+interface ConfigManager
+{
+    /** @return array<string, mixed> */
+    public function load(): array;
+
+    /** @param array<string, mixed> $options */
+    public function save(array $options): bool;
+
+    /** @return array<string, mixed> */
+    public function getSource(): array;
```

Comment:
I don't think these doc block typehints add a lot (I know we've used them in other places as well and I should really go over those and clean up 😅)

Reply:
I would rather leave them. At least, they show that it is expected to be associative arrays :)